news/2025/12/05/ModelingToolkitv11.md

```diff
@@ -0,0 +1,272 @@
+Major props to Aayush Sabarwal for leading much of the development on this release over the path
+9 months. With this, we have a lot to talk about and share. Some of this is a difficult discussion
+which we want to be transparent about, so please read through the entire post.
+
+## Licensing Changes
+
+One significant item to address with this update is the licensing associated with ModelingToolkit.jl.
+Starting with version 11, ModelingToolkit.jl will be structured in a way where some of its dependencies have an AGPL
+license. In short (and will be elaborated on later), ModelingToolkit.jl and SymbolicUtils.jl now include
+an extendable pass system for the compiler, meaning that additional compiler passes can be added into
+the system through downstream dependencies. This is a powerful feature that allows for more extensibility
+and customization of the symbolic transformation process. This is something that was attempted from the
+start, which is why the StructuralTransformation.jl subpackage/submodule was always shipped with
+ModelingToolkit.jl, but the pass system was never modular enough to follow allow for external extensions,
+now it does. 
+
+ModelingToolkitBase.jl is thus an MIT-licensed package with no Julia GPL dependencies and contains all
+of the core functionality of ModelingToolkit.jl. This for example includes the utilities for building
+`System`s, generating code for ODEs, SDEs, etc., and the main symbolic compiler pipeline. 
+ModelingToolkitBase.jl is thus a fully functional symbolic modeling compiler which can take symbolic
+descriptions of systems and generate performant Julia code for them as-is.
+
+With this change, the versions of the symbolic transformation libraries has been moved to
+a separate repository (https://github.com/JuliaComputing/StateSelection.jl) which is licensed under [AGPL](https://www.gnu.org/licenses/agpl-3.0.en.html).
+As AGPL is an [OSI-approved license](https://opensource.org/licenses), this package is open source and released as part of the General
+registry, meaning that ModelingToolkit.jl remains easy to install and use for open source projects.
+However, AGPL does have some implications for commercial use, so please review the license if you
+are considering using ModelingToolkit.jl in a commercial setting. In particular, if you are redistributing
+ModelingToolkit.jl in a shipped binary or web service, then you need to ensure the licensing requirements
+of AGPL are satisfied. It was already the case before that some dependencies of ModelingToolkit.jl and
+Julia itself were GPL dependencies, notably SuiteSparse and FFTW are GPL-licensed libraries and thus
+ModelingToolkit.jl already had some GPL implications. However, with this change, the implications include
+the symbolic transformation libraries specifically for handling acausal models and high-index DAEs.
```

Comment:
```suggestion
of AGPL are satisfied. However, with this change, the implications include
the symbolic transformation libraries specifically for handling acausal models and high-index DAEs.
```
I think this is just muddying the waters and confusing for readers. Having a discussion on the GPL restrictions of base Julia in general is fine, and was clear many people were unaware of this in the Discrouse thread about MTK's licensing, but I don't think this is the place to have such discussions. Let's just stay focused on MTK here.